plugins/http_plugin/include/eosio/http_plugin/beast_http_session.hpp

```diff
@@ -311,27 +311,33 @@ class beast_http_session : public detail::abstract_conn {
          } catch(const fc::exception& e) {
             err_str = e.to_detail_string();
             fc_elog(plugin_state_->logger, "fc::exception: ${w}", ("w", err_str));
-            error_results results{static_cast<uint16_t>(http::status::internal_server_error),
-                                  "Internal Service Error",
-                                  error_results::error_info( e, http_plugin::verbose_errors() )};
-            err_str = fc::json::to_string( results, fc::time_point::now() + plugin_state_->max_response_time );
+            if( is_send_exception_response_ ) {
```

Comment:
Not related to this PR, but I don't think the name `is_send_exception_response_` is meaningful, I would prefer `should_send_exception_response_`.

Reply:
We do somewhat consistently use `is` for bools, but I don't disagree.